testdata/review/pr_description/review

```diff
@@ -0,0 +1,21 @@
+Not necessary but seems ok.
+
+@prr approve
+
+> This is just for testing purposes.
```

Comment:
Would be good to add another test case for review comments interleaving PR summary

Reply:
I have added another test that should cover this.